apps/web/src/lib/components/settings/settings-content.svelte

```diff
@@ -97,36 +100,63 @@
     }
   ];
 
+  let furiganaStyleTooltip = '';
+
   $: verticalMode = writingMode === 'vertical-rl';
+  $: switch (furiganaStyle) {
+    case FuriganaStyle.Full:
+      furiganaStyleTooltip =
+        'Furigana is hidden and displayed on hover. On click the furigana will be permanently displayed.';
+      break;
+    case FuriganaStyle.Toggle:
+      furiganaStyleTooltip = 'Furigana is hidden and can be toggled by clicking.';
+      break;
+    default:
+      furiganaStyleTooltip = 'Furigana is diplayed but grayed out.';
+      break;
+  }
+  $: avoidPageBreakTooltip = avoidPageBreak
+    ? 'Tries to avoid Breaks between Pages.\nMay lead to empty Pages or unused Page Space'
+    : 'Pages will stretch across the availabe Space.\nIncreases Occurences for Page Breaks inside Sentences/Words.\nHas currently a Yomichan Bug';
```

Comment:
"Allow words/sentences to break into different pages\nMay cause Yomichan rendering bug"